src/world/partner/lakilester.c

```diff
@@ -18,6 +18,9 @@ BSS f32 D_802BFF28;
 BSS s32 D_802BFF2C;
 BSS unkPartnerStruct D_802BFF30;
 
+void func_800EF3D4(s16);                               /* extern */
+s32 player_raycast_up_corners(PlayerStatus*, f32*, f32*, f32*, f32*, f32); /* extern */
```

Comment:
Any function declarations from other files like player_raycast_up_corners should go into functions.h and not here in this file